### PR TITLE
(feat) created checkbox component

### PR DESCRIPTION
**Pull request by fakela**
#### What does this PR do
This PR creates checkbox components
#### Description of Task to be completed
Create a reusable check component according to the style guide.
#### How should this be manually tested
By opening up storybook and viewing the checkbox component.
Storybook can be run with the command npm run storybook
#### What are the relevant issues this PR belongs to
#18

### CODE REVIEW

**Comment by fakela** [comment]
@BolajiAyodeji  please review

File: src/components/checkBox/checkBox.stories.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import CheckBox from ".";
+
+
+export default {
+  title: "CheckBox",
+  component: "CheckBox"
+};
+
+
```
Comment (BolajiAyodeji) [inline src/components/checkBox/checkBox.stories.js]: Can you format please, there are some extra spaces here?

File: src/components/checkBox/style.module.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  -webkit-transform: rotate(45deg);
+  -ms-transform: rotate(45deg);
+  transform: rotate(45deg);
+}
+
+.oc_container input:disabled:checked ~ .oc_checkmark {
+  background-color: #96ADDD;
+}
+
+
```
Comment (BolajiAyodeji) [inline src/components/checkBox/style.module.css]: Some extra spaces here too

File: src/components/checkBox/checkBox.stories.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+export const defaultCheckBox = () => <CheckBox/>;
+
+export const CheckBoxChecked = () => <CheckBox checked ></CheckBox>;
+
+export const disabledCheckBox = () => <CheckBox disabled style={{ backgroundColor: "#F0F0F0" }}></CheckBox>;
+
+export const checkedDisabledCheckBox = () => <CheckBox default checked disabled style={{ backgroundColor: "#F0F0F0" }} ></CheckBox>;
```
Comment (BolajiAyodeji) [inline src/components/checkBox/checkBox.stories.js]: I don't think there should be a `checkedDisabled` state, a disabled state communicates when a component or element isn't interactive and should be deemphasized in the UI hence a disabled checkbox should not be checked.

Comment (fakela) [reply src/components/checkBox/checkBox.stories.js]: I just did that based on the style guide

Comment (fakela) [reply src/components/checkBox/checkBox.stories.js]: but it's removed now

File: src/components/checkBox/index.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  return (
+    <label className={_wrapper}>
+      <input
+        type="checkbox"
+        checked={checked}
+        disabled={disabled}
+        onChange={onChange}
+        {...props}
+      />
+      <span className={_className} style={{ ...style }} />
```
[the remark this answers is not in the document]
Comment (fakela) [reply src/components/checkBox/index.js]: yes

**Comment by fakela** [comment]
@BolajiAyodeji I think this is ready for a merge

**Pull request merged**